File: luci-app-dae/luasrc/model/cbi/dae/global.lua
local dae = require "luci.model.dae_tools"
local m, s, o

m = Map("dae", translate("Global Settings"), translate("Configure global settings for DAE."))

local config_file = "/etc/dae/config.dae"

s = dae.init_editor(m, "global")

s:option(Flag, "enabled", translate("Enabled")).rmempty = false

dae.add_editor(s, config_file, "globalconf", translate("Global Configuration"), translate("Correctly configure the include field for separate-config to work, or enter complete configuration here."))

return m

File: luci-app-dae/htdocs/luci-static/resources/view/dae/global.js
'use strict';
'require form';
'require dae.editor as heditor';

/*
 * Global Settings —— 对照 Lua 版 luasrc/model/cbi/dae/global.lua
 * uci 开关（enabled）+ /etc/dae/config.dae 编辑器
 */

return heditor.editorPage({
	key: 'config',
	title: _('Global Settings'),
	description: _('Configure global settings for DAE.'),
	editorTitle: _('Global Configuration'),
	editorDescription: _('Correctly configure the include field for separate-config to work, or enter complete configuration here.'),
	uciSection: function(section) {
		var enable = section.option(form.Flag, 'enabled', _('Start Service'));
		enable.rmempty = false;
	}
});
